backend/buildeR.BLL/Services/Uploads/ImageStorage.cs

```diff
@@ -0,0 +1,94 @@
+﻿using buildeR.BLL.Interfaces.Uploads;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace buildeR.BLL.Services.Uploads
+{
+    public class ImageStorage : IImageStorage
```

Comment:
Could you rename it to FileStorage that have an ability to add or remove a file from the disk? In the future it can be not the only images